.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+rust-peer/*             @libp2p/rust-libp2p-maintainers
+go-peer/*               @libp2p/go-libp2p-maintainers
+packages/frontend/*     @libp2p/js-libp2p-chainsafe-maintainers
```

Comment:
I think this  needs to be changed to `js-peer` and also the codeowner is `@libp2p/js-libp2p-dev` see the [js-libp2p CODEOWNERS file](https://github.com/libp2p/js-libp2p/blob/master/CODEOWNERS) for reference.

Reply:
```suggestion
js-peer/*     @libp2p/js-libp2p-dev 
```